### PR TITLE
Improve the formatting of a figure for the inner core example.

### PR DESCRIPTION
**Pull request by bangerth**
Also trim whitespace around some of the .png files. This is a follow-up to @jdannberg's #1121.

### CODE REVIEW

File: doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out [hunk 1]
```diff
@@ -5,10 +5,9 @@ subsection %%\hyperref[parameters:Initial_20conditions]{Initial conditions}%
...
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Thermal expansion coefficient in initial temperature scaling]{Thermal expansion coefficient in initial temperature scaling}% = 3e-5%% \index[prmindex]{Thermal expansion coefficient in initial temperature scaling} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Thermal expansion coefficient in initial temperature scaling} %
     set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Reference temperature]{Reference temperature}%             = 1600%% \index[prmindex]{Reference temperature} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Reference temperature} %
-    set %%\hyperref[parameters:Initial conditions/S40RTS perturbation/Remove temperature heterogeneity down to specified depth]{Remove temperature heterogeneity down to specified depth}% = 200000%% \index[prmindex]{Remove temperature heterogeneity down to specified depth} \index[prmindexfull]{Initial conditions!S40RTS perturbation!Remove temperature heterogeneity down to specified depth} %
```
Comment (jdannberg) [inline doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out]: Why are there changes to this file?

**Comment by jdannberg** [comment]
The formatting looks good, but there is this one test that fails, which has nothing to do with this pull request, but was already failing for #1120.

File: doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out (hunk 1 above)
Comment (bangerth) [inline doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out]: Good question. I missed that I accidentally included this file. But the changes are correct: a previous patch updated the `S20RTS.part.prm` file but not the `S20RTS.part.prm.out` file. The changes above just bring them back in synch again.

Comment (jdannberg) [inline doc/manual/cookbooks/initial-condition-S20RTS/S20RTS.part.prm.out]: Okay, good. I guess we can merge this one then, but we should look into what happened in #1120. As you reviewed that, do you have an idea what the problem is?

**Comment by bangerth** [comment]
Yes, I've ignore that one test in merging some recent commits. We'll have to figure out what is happening on @tjhei 's machine.

**Pull request merged**